.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```

Comment:
Should this not be yarn?

Reply:
<img width="638" alt="Screenshot 2021-12-17 at 7 59 23 PM" src="https://user-images.githubusercontent.com/69431456/146559463-03a54fd3-1b47-441b-9203-fef3f931a3ec.png">

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem

npm is for both npm and yarn registries 😄